src/pudl/transform/eia861.py

```diff
@@ -448,17 +448,24 @@
 
 CUSTOMER_CLASSES = [
     "commercial",
+    "dircnct",  # previously direct_connection
     "industrial",
     "other",
     "residential",
     "total",
     "transportation"
 ]
 
-FUEL_CLASSES = [
+TECH_CLASSES = [
     'pv',
     'wind',
-    'chpcogen',
+    'chpcogen',  # previously chp_cogen
+    'combturb',  # previously combustion_turbine
+    'fcell',  # previously fuell_cell
+    'hydro',
+    'ice',  # previously internal_combustion
+    'steam',
+    'storage',
```

Comment:
These are the non-underscore names of the tech and customer classes we need in order to cleanly grab them as prefixes. I assume we'll want the names to be their longer versions in the final tables so that they are more readable, and thus we should probably store their longer names in connection with these short names (maybe by converting these lists to dictionaries) and do a `replace` or something of the sort in the tables.

Reply:
sounds reasonable!